low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -162,11 +159,11 @@ def make_response(self, component_details, component, component_type,
             del component_details['severity']
             if self.SEVERITY == "warning":
                 del component_details['description']
-        elif component == "hw":
+        elif component_type != "platform_sensor":

```

Comment:
hw should not be removed, because type sw also there which can landup here. So keep
> component == "hw" && component_type != "platform_sensor":

Reply:
Done.